### PR TITLE
Brings Grakn 1.8 Support

### PR DESCRIPTION
**Pull request by sorsaffari**
## What is the goal of this PR?
This PR brings the Grakn 1.8 support to Workbase.

## What are the changes implemented in this PR?
resolves https://github.com/graknlabs/workbase/issues/290

Following https://github.com/graknlabs/client-nodejs/commit/b85668f7063b2313ed02ab35cd0922e67d1f9f4b

- `dataType` to `valueType`
- `datatype` to `value`
- `date` to `datetime`
- removing all references to `isImplicit`
- bump `client-nodejs` and `grakn-core` dependencies

### CODE REVIEW

File: src/renderer/components/SchemaDesign/SchemaUtils.js [hunk 1]
```diff
@@ -27,15 +27,15 @@ export async function loadMetaTypeInstances(graknTx) {
     .then(labels => labels.filter(l => l !== 'entity')
       .concat()
       .sort());
-  metaTypeInstances.relations = await Promise.all(rels.map(async type => ((!await type.isImplicit()) ? type.label() : null)))
+  metaTypeInstances.relations = await Promise.all(rels.map(async type => type.label()))
```
Comment (flyingsilverfin) [inline src/renderer/components/SchemaDesign/SchemaUtils.js:30]: yaohoohoooo

File: src/renderer/components/Visualiser/TopBar/GraqlEditor/GraqlCodeMirror.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ CodeMirror.defineSimpleMode('graql', {
   start: [
     { regex: /#.*/, token: 'comment' },
     { regex: /".*?"/, token: 'string' },
-    { regex: /(match|insert|delete|select|isa|sub|plays|relates|datatype|abstract|has|value|id|of|limit|offset|order|by|compute|from|to|in|aggregate|label|get|using|where)(?![-a-zA-Z_0-9])/, // eslint-disable-line max-len
+    { regex: /(match|insert|delete|select|isa|sub|plays|relates|value|abstract|has|value|id|of|limit|offset|order|by|compute|from|to|in|aggregate|label|get|using|where)(?![-a-zA-Z_0-9])/, // eslint-disable-line max-len
```
Comment (flyingsilverfin) [inline src/renderer/components/Visualiser/TopBar/GraqlEditor/GraqlCodeMirror.js:10]: we don't have aggregate anymore
`value` is in this list twice
 `select` isn't a keyword anymore (?)
`label` isn't a keyword anymore (we use `type`)

File: test/integration/SchemaDesign/SchemaHandler.test.js [hunk 3]
```diff
@@ -44,7 +44,7 @@ describe('Actions', () => {
     let graknTx = await graknSession.transaction().write();
     const schemaHandler = new SchemaHandler(graknTx);
 
-    await schemaHandler.defineAttributeType({ attributeLabel: 'name', dataType: 'string' });
+    await schemaHandler.defineAttributeType({ attributeLabel: 'name', valueType: 'string' });
```
Comment (flyingsilverfin) [inline test/integration/SchemaDesign/SchemaHandler.test.js:47]: why ared the mocked value types `String` and this `string`?

File: src/renderer/components/Visualiser/TopBar/GraqlEditor/GraqlCodeMirror.js (hunk 2 above)
Comment (sorsaffari) [reply src/renderer/components/Visualiser/TopBar/GraqlEditor/GraqlCodeMirror.js]: addressed in https://github.com/graknlabs/workbase/pull/291/commits/fa0a2c1886642232b97fbaaf24ddec210c43a3e1

keywords are reduced to only those that are used in queries supported by Workbase (currently, `match get` and computer path`)

File: test/integration/SchemaDesign/SchemaHandler.test.js (hunk 3 above)
Comment (sorsaffari) [reply test/integration/SchemaDesign/SchemaHandler.test.js:47]: good catch, josh!
resolved in https://github.com/graknlabs/workbase/pull/291/commits/dc170df40bb42ce67da51f5b885a710e1cedecfd

**Pull request merged**